components/BottomNavigation/src/MaterialBottomNavigation.bundle/Resources/en.lproj/MaterialBottomNavigation.strings

```diff
@@ -0,0 +1,3 @@
+/* Accessibility hint preposition for item number i of n. */
+"MaterialBottomNavigationItemCountAccessibilityHint" = "of";
```

Comment:
I believe this is insufficient.  I seem to recall certain languages may reorder this.  So were we might say "2 of 5", another language might be "of 5 item 2".  So the order of the parameters may be reversed.

Reply:
+1. I believe the preferred approach is to keep the entire phrase together for context, something like,
```
/* Accessibility hint for the item number. Used when blah blah... */
"tab %1$d of %2$d"
```